api/metrics.go

```diff
@@ -401,6 +440,24 @@ func recordGauges(c *gin.Context) {
 			}
 		}
 
+		// idle, available, busy, error counts
+		for _, worker := range workers {
```

Comment:
Could this entire loop be brought into the already existing loop above? Under https://github.com/go-vela/server/pull/772/files#diff-ffb09227a043a8b68ade16603539951b8e216f71125de14b6919ab63fee86bc5R433-R435 perhaps?